openaps_reports.py

```diff
@@ -98,3 +105,12 @@ def recent_dose(self):
                 return self._read_json(Settings.SET_DOSE, [])
 
         return []
+
+    def data_age(self):
+        fn = self._get_report_path(Settings.PREDICT_GLUCOSE)
```

Comment:
I actually tried that already. The issue is that (at least in my config), the normalize_history.json file is rebuilt every single time, irrespective of whether the data is fresh. So the most recent file is always current.

The same problem would occur if someone created a lockfile in the directory, for example.

My intention is to show the last successful read from the pump - rather than the last successful run of the loop.

I'm wondering if we should simply read the 'read_clock' file. However, it's always possible that the clock time could have been read, but glucose data failed to read.

What do you think's best here?